src/website/src/app/documentation/get-started/get-started.component.html

```diff
@@ -20,31 +20,34 @@ <h3 id="designResources">Using Clarity Design Resources</h3>
     <div class="design-assets">
         <div>
             <h4>Primary Font</h4>
-            <a class="btn btn-primary btn-sm" href="https://github.com/chrismsimpson/Metropolis/archive/r8.zip" target="_blank" rel="noopener">
+            <a class="btn btn-primary btn-sm" href="https://github.com/chrismsimpson/Metropolis/archive/r8.zip" target="_blank" rel="noopener" (click)="gaClick('font-metropolis')">
                 Metropolis (R8)
             </a>
         </div>
         <div>
-            <h4>Sketch Libraries</h4>
-            <a [href]="sketchBaseUrl + '/light/clarity-library-light-' + sketchVersion + '.sketch'" target="_blank"
-               class="btn btn-primary btn-sm" rel="noopener">UI Library - Light ({{sketchVersion}})
+            <h4>Figma Libraries</h4>
+            <a [href]="lightFigma" target="_blank" (click)="gaClick('figma-light')"
+                class="btn btn-primary btn-sm" rel="noopener">UI Library - Light ({{sketchVersion}})
```

Comment:
should there be a figmaVersion or is this just overloading the the sketchVersion since they are still the same?

Reply:
This is just overloading since they stay in sync